lib/era_ja/conversion.rb

```diff
@@ -3,6 +3,8 @@
 
 module EraJa
   module Conversion
+    class DateOutOfRangeError < StandardError; end
```

Comment:
I think it is better to place the error class to EraJa :: DateOutOfRangeError instead of EraJa::Conversion::DateOutOfRangeError.

Because user do not see EraJa::Conversion.
Do you think about it?

Reply:
@tomiacannondale 

Thank you for your comment.

> I think it is better to place the error class to EraJa :: DateOutOfRangeError instead of EraJa::Conversion::DateOutOfRangeError.
> 
> Because user do not see EraJa::Conversion.
> Do you think about it?

When creating this PR, I was wondering which one to define `EraJa::Conversion` or `EraJa::Conversion::DateOutOfRangeError`. 

But, as you comment, I think `EraJa::Conversion` is better.
I will fix the code. Thanks.

Reply:
> When creating this PR, I was wondering which one to define `EraJa::Conversion` or `EraJa::Conversion::DateOutOfRangeError`.
> 
> But, as you comment, I think `EraJa::Conversion` is better.
> I will fix the code. Thanks.

Sorry, `EraJa::Conversion` is typo.. I meant to write `EraJa::DateOutOfRangeError`.